src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -380,23 +381,23 @@ def run_evals(
 
     async def _arun_eval(
         payload: RunEvalsPayload,
-    ) -> Tuple[EvaluatorIndex, RowIndex, Label, Explanation]:
-        label, explanation = await payload.evaluator.aevaluate(
+    ) -> Tuple[EvaluatorIndex, RowIndex, Label, Score, Explanation]:
+        label, score, explanation = await payload.evaluator.aevaluate(
```

Comment:
Kinda confused why classify has to be touched. Can't it be done independently?

Reply:
This is a tweak to `run_evals`, which currently lives inside the `classify` sub-module.

Reply:
I see, nice.